dbms/src/Storages/DeltaMerge/DeltaMergeStore_Ingest.cpp

```diff
@@ -1226,7 +1226,7 @@ UInt64 DeltaMergeStore::ingestSegmentsFromCheckpointInfo(
 
     auto restored_segments = checkpoint_info->getRestoredSegments();
     auto updated_segments = ingestSegmentsUsingSplit(dm_context, range, restored_segments);
-    auto estimated_bytes = 0;
+    size_t estimated_bytes = 0;
```

Comment:
`auto` in here unexpectedly lead to negative value because the compiler use `int32` for the auto. And collecting the bytes of restored_segment could larger than 2GiB.